web/plugs/authorization.ex

```diff
@@ -75,12 +75,12 @@ defmodule Cog.Plug.Authorization do
   # in the future. For example, this would allow users to update
   # related data with paths such as `/v1/users/:user_id/profiles/:id`.
   defp self_updating?(opts, conn) do
-    if Keyword.get(opts, :allow_self_updates, false) == true do
-      conn.private.phoenix_action in [:update, :show] and
-      (conn.assigns.user.id == conn.params["id"] or
-       conn.params["id"] == "me")
-    else
-      false
+    case Keyword.get(opts, :allow_self_updates, []) do
+      [] -> false
+      actions ->
```

Comment:
This all might read easier if we changed it from `:allow_self_updates` to something like `:self_updating_actions`. The former "sounds" Boolean, while the latter is more suggestive of a list.